src/03-control-flow.js

```diff
@@ -146,7 +194,11 @@ function calculateLetterGrade(grade) {}
  *  animalCounts("pig", 1);
  *  //> "1 pig"
  */
-function animalCounts(animal, numberOfAnimals) {}
+function animalCounts(animal, numberOfAnimals) {
+  if (numberOfAnimals >= 2){
+    animal += 's'}
```

Comment:
This is really clever!  I like how you change the animal string. Does this change the original string?